uwblueprint/childrens-aid-society
# Create add child page with form selection component

## Notion ticket link
[Implement ‘Add Child’ page structure](https://www.notion.so/uwblueprintexecs/Implement-Add-Child-page-structure-1d94c12e15a54442b96013ec87b6b09b)


## Implementation description
* Following a similar approach to the`Stepper` component, I created a `FormSelector` component that updates and reflects the `AddChildPage` form state.
* Render `ChildInformationForm`, `ChildProviderForm`, or `SchoolDaycareForm` dependent on `AddChildPage` state



## Steps to test
1. [Link to Figma](https://www.figma.com/file/1obAgfYZ639hpKXV1ZP7hJ/3.-FeatureSets-%3E-Case-intake?node-id=1391%3A774867)
2. Verify that add child page header matches Figma design
3. Verify the "Back to case individuals" button routes back to `IntakePage` 



## What should reviewers focus on?
* I'm not too sure if using `Link` is the correct choice for the form selector component. Would it be better to use `Button` and give it custom styling?


## Checklist
